tests/fips/fips_setup.pm

```diff
@@ -29,6 +30,14 @@ sub run {
         assert_script_run("grep '^1\$' /proc/sys/crypto/fips_enabled");
         record_info 'Kernel Mode', 'FIPS kernel mode (for global) configured!';
 
+        # Stop packagekitd
+        if (is_sle('12+')) {
+            quit_packagekit;
+        }
+        else {
+            assert_script_run "chmod 444 /usr/sbin/packagekitd";
+        }
+
```

Comment:
Ben,
Why not use quit_packagekit on sles12+?

Reply:
It has already done everything there: :)
  script_run("systemctl mask packagekit; systemctl stop packagekit; while pgrep packagekitd; do sleep 1; done");

Reply:
I just check that lib/migration use this style, probably it could not impact the maintenance job group. Do you think this is feasible?
or maybe just one line `quit_packagekit;` is also okay.

Reply:
chmod will not stop packagekitd service there, it can only make sure it will not start in next system up

Reply:
@rfan1 nice comment. After discussion offline, I will change to `quit_packagekit;` only, thanks.

Reply:
Done